RInAppMessaging/Classes/Helpers/UITestHelper.swift

```diff
@@ -0,0 +1,32 @@
+internal struct UITestHelper {
```

Comment:
I'm not keen on this being in the SDK folder's source.

Reply:
I know but to be honest I think this is the least intrusive way of configuring the SDK for UI Tests.
We could use some external tool to mock the endpoints but that's much more complex solution.

Reply:
I think it's a common pattern on Android to use MockWebServer for running (UI) tests and on iOS there's the possibility of using https://github.com/httpswift/swifter - could be a better approach for us?

Reply:
I'd be more comfortable if this struct was namespaced under `Environment`

Reply:
I removed `UITestHelper` completely. The SDK is now entirely UITest-agnostic